tests/test_handlers.py

```diff
@@ -0,0 +1,172 @@
+                                 '{bold_yellow}\d{{4}}-\d{{2}}-\d{{2}} \d{{2}}:\d{{2}}:\d{{2}},\d+ '
+                                 'FRONTIER_START   info message{reset}\n'.format(
+                                     bold_yellow=re.escape(colors['bold_yellow']),
+                                     reset=re.escape(colors['reset'])))
+
+    def test_handler_events_level(self):
+        self.logger.debug('debug message', extra={'event': 'FRONTIER_START'})
+        self.logger.info('info message', extra={'event': 'FRONTIER_STOP'})
+        self.assertRegexpMatches(self.logger_output.getvalue(),
+                                 '{bold_yellow}\d{{4}}-\d{{2}}-\d{{2}} \d{{2}}:\d{{2}}:\d{{2}},\d+ '
+                                 'FRONTIER_STOP    info message{reset}\n'.format(
+                                     bold_yellow=re.escape(colors['bold_yellow']),
+                                     reset=re.escape(colors['reset'])))
+
+    def test_handler_events_filter(self):
+        self.logger.info({'message1': 'info', 'message2': 'message', 'event': 'FRONTIER_START'})
+        log = self.logger_output.getvalue()
+        pattern1 = '{bold_yellow}\d{{4}}-\d{{2}}-\d{{2}} \d{{2}}:\d{{2}}:\d{{2}},\d+ ' \
+                   'FRONTIER_START   {logged_message}{reset}\n'.format(bold_yellow=re.escape(colors['bold_yellow']),
+                                                                       logged_message='message info',
+                                                                       reset=re.escape(colors['reset']))
+        pattern2 = '{bold_yellow}\d{{4}}-\d{{2}}-\d{{2}} \d{{2}}:\d{{2}}:\d{{2}},\d+ ' \
+                   'FRONTIER_START   {logged_message}{reset}\n'.format(bold_yellow=re.escape(colors['bold_yellow']),
+                                                                       logged_message='info message',
+                                                                       reset=re.escape(colors['reset']))
+        assert re.match(pattern1, log) is not None or re.match(pattern2, log) is not None
+
+
+class TestHandlerConsole(BaseTestHandlers):
+    handler = CONSOLE
+
+    def test_handler_color_based_on_level(self):
+        self.logger.debug('debug message')
+        self.logger.info('info message')
+        self.logger.error('error message')
+        self.assertEqual(self.logger_output.getvalue(),
+                         '{white}[frontera] debug message{reset}\n'
+                         '{green}[frontera] info message{reset}\n'
+                         '{red}[frontera] error message{reset}\n'.format(white=colors['white'],
+                                                                         green=colors['green'],
+                                                                         red=colors['red'],
+                                                                         reset=colors['reset']))
+
+
+class TestHandlerConsoleManager(TestHandlerConsole):
```

Comment:
@sibiryakov I totally get what you're trying to say. I myself went to through a similar dilemma when I wrote these tests, whether this code was really useful. The reason why I wrote them anyways was because I got a chance to learn. I don't mind removing the code altogether and also the tests that I've added.

>  Please check that _MANAGER, _BACKEND, _DEBUGGING and _EVENTS is used somewhere.

I had checked this when I had written the tests. This code actually should have been removed as part of commit  https://github.com/scrapinghub/frontera/commit/d4d668be50c05aed583984f95d56c799ac0e56df where `FrontierLogger` and `EventLogManager` were removed. This code doesn't make much sense without the `FrontierLogger` and `EventLogManager`. I think this code should be removed and  while adding docs we can provide examples of different use cases of the  `ColorFormatter`.

> Color logging is nice feature, but useful only when running in CMD line. JSON formatter is useful for sure.

I like them too and I think we should keep them. They can be useful.

>  Filters? Probably yes, mainly to filter events by type, or URLs

The Filters look like they were designed specially to work along with  `ColorFormatter`. If we are keeping `ColorFormatter` than I think we should keep them too. 

> Redis? Do you think it's a good idea to store log in redis?

I'm not very sure if the redis handler is useful. May be helpful in a distributed setup so that all logs can be retrieved from one common place. But personally I don't think the `RedisListHandler` in `frontera`  is very useful. Looks like it was copied from https://github.com/jedp/python-redis-log and slightly modified. I think the code at there is much better and if we ever feel the need for redis lets use the library rather. I'd say lets remove the `RedisListHandler`.
 
Can you reply with your second thoughts. So that I know exactly what to do.

>  I don't want to demotivate you, but this is how it is.

I don't mind any suggestions. As long as you review my code regularly expect a lot more contributions from me :)